### PR TITLE
RCLL-25 Scaffold in some API's and build first screen (enter recall date) - this is just a POC at the moment

### PR DESCRIPTION
**Pull request by yasinmustafa**
Screenshot 2024-06-18 at 13 11 40

### CODE REVIEW

File: assets/images/prisoner-profile-image.png
Comment (joelstobart-moj) [inline assets/images/prisoner-profile-image.png:1]: Space alien

File: server/routes/apiRoutes.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import PrisonerService from '../services/prisonerService'
+
+const placeHolderImage = path.join(process.cwd(), '/assets/images/prisoner-profile-image.png')
+export default class ApiRoutes {
+  constructor(private readonly prisonerService: PrisonerService) {}
+
+  public personImage: RequestHandler = async (req, res): Promise<void> => {
+    const { nomsId } = req.params
+    return this.prisonerService
+      .getPrisonerImage(nomsId, res.locals.user.token)
```
Comment (carloveo-moj) [inline server/routes/apiRoutes.ts:12]: does this need to be a system token?

**Review by joelstobart-moj — DISMISSED** [review]
LGTM

File: server/views/pages/recallEntry/enter-recall-date.njk [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      text: "For example, 27 3 2022"
+    }
+  }) }}
+
+  <div class="govuk-button-group">
+    {{ govukButton({
+      text: "Continue",
+      type: submit,
+      preventDoubleClick: true,
+      attributes: { 'data-qa': 'submit-nter-recall-date' }
```
Comment (carloveo-moj) [inline server/views/pages/recallEntry/enter-recall-date.njk:27]:
```suggestion
      attributes: { 'data-qa': 'submit-enter-recall-date' }
```

**Review by carloveo-moj — APPROVED** [review]
LGTM

**Pull request merged**